db/migrate/20230725152007_create_users.rb

```diff
@@ -0,0 +1,9 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :username, null: false
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
Please consider adding a password field to the Users table, we will need it for authentication.

Reply:
That would be done during the devise setup